package.json

```diff
@@ -8,11 +8,18 @@
     "@testing-library/user-event": "^13.5.0",
     "echarts": "^5.4.0",
     "echarts-for-react": "^3.0.2",
+    "leaflet": "^1.9.2",
```

Comment:
Make sure that next time you uninstall the map dependencies! These aren't needed in your pull request.